docs/content/dagster-cloud/deployment/agents/docker.mdx

```diff
@@ -102,6 +102,41 @@ height={320}
 
 ---
 
+## Granting Registry Access to Docker Agent
+
+The Docker agent runs Docker operations from within the `dagster-cloud-agent` container, including pulling user code images and running them as new containers. The `dagster-cloud-agent` container requires access to credentials in order to perform these operations. Depending on the registry provider you are using, authentication may look different. Often, credentials are stored in the Docker config file; `~/.docker/config.json`.
+
+In some cases, you will need to mount this file into the agent by specifiying the additional volume mount:
+
+```shell
+  --volume ~/.docker:/root/.docker
+```
+
+For example:
+
+```shell
+docker run \
+  --network=dagster_cloud_agent \
+  --volume $PWD/dagster.yaml:/opt/dagster/app/dagster.yaml:ro \
+  --volume /var/run/docker.sock:/var/run/docker.sock \
+  --volume ~/.docker:/root/.docker \
+  --restart on-failure \
+  -it docker.io/dagster/dagster-cloud-agent:latest \
+  dagster-cloud agent run /opt/dagster/app
+```
+
+### Using Google Container Regsitry
+
+The easiest way to grant the agent container access to a GCR repository is to use the [standalone GCR credential helper](https://github.com/GoogleCloudPlatform/docker-credential-gcr) to create an access token:
+
+    # Replace $YOUR_REGISTRY with the GCR registry you are using, e.g. https://gcr.io
```

Comment:
This should be formatted the same way as other code blocks, like on line 117. Without it, the formatting looks funky when the site is built + rendered.